docker/README.md

```diff
@@ -0,0 +1,40 @@
+# Google Click to Deploy containers
+
+## About
+
+This directory hosts the source code of Google Click to Deploy containers images
+available through Google Cloud Platform Marketplace.
+
+## Disclaimer
+
+This is not an officially supported Google product.
+
+## Docker images
+
```

Comment:
Below you can find the list of repositories of Google Click to Deploy container images that were not migrated to this repositories yet. This list is only for your reference and once the migration finishes this list will be empty.